pkg/telefork/telefork.go

```diff
@@ -0,0 +1,5 @@
+// Package telefork contains basic telefork client and exposes tracing presend handler.
+//
+// Package telefork contains basic telefork client and exposes tracing presend handler.
+// This package is used in bootstrap CLIs for basic telemetry.
+package telefork
```

Comment:
Its too bad this package can't just go in telefork, but it's not a public package and this is
.

Reply:
I agree. If I was going to spend more time on this now, I'd probably make it a generic HTTP thing instead, and configure it in bootstrap. That way there would be a good separation between things.

That said, it seems too much work at this point. I'd rather get more detailed telemetry out of CLIs, if I can choose.

Reply:
Totally agree